lib/useOnyx.ts

```diff
@@ -137,6 +144,12 @@ function useOnyx<TKey extends OnyxKey, TReturnValue = OnyxValue<TKey>>(key: TKey
     // in `getSnapshot()` to be satisfied several times.
     const isFirstConnectionRef = useRef(true);
 
+    // Indicates if the hook is connecting to a Onyx key.
```

Comment:
```suggestion
    // Indicates if the hook is connecting to an Onyx key.
```

Reply:
I was going to apply suggestion but you already merged it 😄

Reply:
Feel free to do it in some future PR, sorry about this